common/primitives/task_queues.go

```diff
@@ -26,5 +26,10 @@ package primitives
 
 // These are task queue names for internal task queues.
 const (
-	PerNSWorkerTaskQueue = "temporal-sys-per-ns-tq"
+	DefaultWorkerTaskQueue = "default-worker-tq"
+	PerNSWorkerTaskQueue   = "temporal-sys-per-ns-tq"
+
+	MigrationActivityTQ           = "temporal-sys-migration-activity-tq"
+	AddSearchAttributesActivityTQ = "temporal-sys-add-search-attributes-activity-tq"
```

Comment:
For the sake of "beauty and symmetry", yes. But really the traffic on this TQ is so small, so it doesn't matter.